### DISCUSSION TITLE
Why does this project have additional categories?

### DISCUSSION THREAD

**Discussion by nemanjaglumac** [AaruBama/memestr#63] [Q&A] [answered]
The name of the app was is simple at to the point - **Memestr**. You come here for memes.
So I was wondering why did you decide to offer additional categories? I think it dilutes the main idea and adds unnecessary clutter. I can elaborate this in a future comment(s), but curious to hear author's reasoning first.

And especially why NSFW?
This in particular doesn't seem to belong here. People of all ages enjoy memes. So why offer unfiltered adult content to people who shouldn't even see it? And all that without any warning or at least "click on a box that someone is 18".

There is more than enough NSFW content on internet. Too much, actually.
Whoever wants to find it will find it. Just get rid of it. Keep it light, keep it funny. Make it accessible to all age groups.

**Comment by AaruBama** [comment, marked as answer]
> So I was wondering why did you decide to offer additional categories?

Memestr is memes + trending content of different sorts.
Default feed is very diverse. People like if they have interest-centric content they can look at. That's why the multiple categories. They don't dilute, they separate the type of content people want to look at. Anything they find interesting or funny or even unique and they wanna share it with the community, they can share and tag it to a certain type so other people can enjoy it too.'

> And especially why NSFW? This in particular doesn't seem to belong here.

**An 18+ verification box should definitely be up there**.  Honestly, a lot of people like to make NSFW memes too. NSFW is not exactly a category meant for 18+ content but for naughty memes, and communication to the meme community on memestr. We can discuss more about making this category more meme-friendly than porn-friendly.

> Whoever wants to find it will find it. Just get rid of it. Keep it light, keep it funny. Make it accessible to all age groups.

Again, the NSFW shouldn't have completely nude content is something I very much agree with. Getting rid of it would not be solving the problem but just postponing it.
